evident/tests/support_files/simulated_data_setting.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## This notebook is to simulate the data (mapping files and alpha diversity) for testing"
```

Comment:
This notebook is to simulate the data (mapping files and alpha diversity) for testing -> This notebook is to generate the simulated data for testing.

Reply:
done